### DISCUSSION TITLE
[COMUNICAZIONE IMPORTANTE] adeguamento dei gestionali dello stato civile per nuova numerazione degli atti digitali prodotti da ANSC

### DISCUSSION THREAD

**Discussion by AlessandriniPCM** [italia/ansc#1373] [Announcements]
Gentilissimi, 

il 24 ottobre 2023 il sistema ANSC – archivio nazionale informatizzato dei registri dello stato civile, definito dal DM del 22 ottobre 2022 del Ministero dell’interno, inizierà ad essere utilizzato per la registrazione di atti di stato civile digitale validi ad ogni effetto di legge, da parte di un gruppo di quattro comuni, cui seguirà un secondo gruppo di altri quattro a novembre. 

Si dà così avvio alla fase di adozione controllata. Tale fase è caratterizzata da una situazione ibrida nella quale i comuni che hanno aderito ad ANSC formano atti di stato civile digitali (comuni digitali) e i comuni che non hanno ancora aderito ad ANSC (comuni analogici) continuano a formare atti di stato civile cartacei. In tale situazione i **gestionali dello stato civile** dei comuni analogici devono essere adattati e predisposti – qualora non si fosse già provveduto - per ricevere e gestire la nuova numerazione degli atti digitali che potranno essere inviati, ad esempio, per la trascrizione o per ogni altro adempimento, dai comuni digitali verso i comuni analogici. 

Si ricorda infatti che il passaggio ad un unico registro nazionale informatizzato, comporta un nuovo sistema di numerazione degli atti dello stato civile, costituito dall’anno di riferimento, dal numero progressivo nazionale, dal numero progressivo comunale e dal codice ISTAT identificativo del comune [anno-n.nazionale-n.comunale-codistat], [ES. 2023-1002-10-043023]. 

Nonostante la gradualità dell’adesione da parte dei comuni al nuovo sistema ANSC, tenuto conto dell’imminente avvio dell’adozione controllata, si rende necessario che **tutti gli enti locali** adeguino i sistemi gestionali dello stato civile alla nuova numerazione. I comuni dovranno aggiornare il loro software di stato civile per consentire l’acquisizione e la gestione sia dell'attuale numerazione  (numero/parte/serie), sia della nuova numerazione degli atti dello stato civile digitali sopra esemplificata. 

Ci permettiamo di suggerire, così come emerso dal confronto con alcune SH in sperimentazione, una possibile, temporanea ma veloce soluzione, che consiste nell’utilizzo, ove fattibile per le caratteristiche dei gestionali, dell’attuale campo ‘numero’ lasciando vuoti i campi ‘parte’ e ‘serie’.   

Nel metterci a disposizione per una call nel caso fossero necessari ulteriori chiarimenti, anticipiamo che nei prossimi giorni verrà emanata una circolare del Ministero dell’interno, sulla gestione dell’ibrido contenente indicazioni, anche su questo particolare aspetto. Ci preme con questa comunicazione anticiparne parte del contenuto in modo che, sia le software house sia i comuni, possano beneficiare di un tempo maggiore per adeguare i gestionali e le procedure correlate come sopra descritto. 

Ringraziamo per la consueta e fattiva collaborazione.

**Comment by MoniaPedrazzoli** [comment]
Buongiorno, vista la proposta di utilizzare l'attuale campo 'numero', lasciando vuoti i campi ‘parte’ e ‘serie’, per indicare il riferimento all'atto originale digitale, è quindi possibile inserire solo il 'n.nazionale' e non la stringa completa [anno-n.nazionale-n.comunale-codistat]? Questo significherebbe riportare anche nella stampa dell'atto trascritto il solo elemento del numero progressivo nazionale quale identificativo dell'atto originale digitale.

**Comment by AlessandriniPCM** [comment]
Buongiorno, 
la parte 'numero' dell'attuale numerazione deve contenere l'intera stringa [anno-n.nazionale-n.comunale-codistat] della nuova numerazione in modo che l'atto venga identificato univocamente e sempre con lo stesso identificativo. Grazie e buona giornata.

**Comment by rinonasti** [comment]
Nei servizi ANPR 3002 verrà inserito il tag ATTO ANSC ?

**Comment by mttfranci** [comment]
Buon pomeriggio @rinonasti .

L'intervento è già presente sul fronte di ANPR a partire dalla versione 8.0 annunciata con l'avviso : 
https://github.com/italia/anpr/issues/3883 

Trovate la documentazione nell'area preposta del del portale : 
https://www.anagrafenazionale.interno.it/area-tecnica/documentazione-tecnica/

I nuovi tag si attivano solo indicando il parametro `codDestintario` = `ANPR05`, come da [Change Management ANPR](https://github.com/italia/anpr/blob/master/src/change-management/change-management.md).

La release 8.0 include anche la 7.0 (che si attiva invece con `codDestintario` = `ANPR04` e contiene il dettaglio dell'atto digitale) : 
https://github.com/italia/anpr/issues/3625

Inoltre, come ovvio, non saranno restituiti atti digitali in produzione fino a che i comuni non inizieranno a registrarli.

Siamo a disposizione per qualsiasi chiarimento.

Saluti
